app/frontend/src/store/modules/form.js

```diff
@@ -166,13 +170,21 @@ export default {
     },
     async updateForm({ state, dispatch }) {
       try {
+        const emailList =
+          state.form.sendSubRecieviedEmail &&
+            state.form.submissionReceivedEmails &&
+            Array.isArray(state.form.submissionReceivedEmails)
```

Comment:
Could be in there, but it's ok to leave out that additional condition, because if I set a blank array, then it would be [] anyways, which is what the else clause sets it to if the conditional fails (sets it to [])